.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -50,7 +50,7 @@ body:
     required: true
 - type: dropdown
   attributes:
-    label: What arch are you using?
+    label: What architecture are you using?
```

Comment:
if you are going to update this, then specify "CPU architecture" instead.

Reply:
done